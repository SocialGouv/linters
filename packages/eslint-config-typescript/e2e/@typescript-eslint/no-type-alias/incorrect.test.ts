import {
  fixtures,
  fixtureToTestCase,
  ruleTest,
  TIMEOUT,
} from "e2e/rule_tester";
import { basename } from "path";

describe(`@typescript-eslint/${basename(__dirname)} off`, () => {
  test.each(fixtures(__dirname, /^incorrect$/).map(fixtureToTestCase))(
    "should invalidate : %s",
    ruleTest,
    TIMEOUT
  );
});
