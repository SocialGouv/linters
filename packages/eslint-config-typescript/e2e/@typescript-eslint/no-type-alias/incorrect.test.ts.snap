// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@typescript-eslint/no-type-alias incorrect 1`] = `""`;
